<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Data Packert Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="moudleComBox">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>25</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>ARP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TCP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UDP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>IP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ICMP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ALL</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Amount:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pktAmountLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="runButton">
          <property name="text">
           <string>Grap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="minimumSize">
         <size>
          <width>646</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Source IP</string>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Source MAC</string>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Dest IP</string>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Dest MAC</string>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Protocol</string>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Length</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Others</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Choose  Network Driver</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="serchButton">
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListView" name="listView"/>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>28</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Network Driver Information:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="addrEdit"/>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="outputLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Data Packet:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QTextEdit" name="textEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
